app/controllers/spree/admin/orders_controller.rb

```diff
@@ -99,11 +100,16 @@ def invoice
       end
 
       def print
-        if OpenFoodNetwork::FeatureToggle.enabled?(:invoices)
-          @order = @order.invoices.find(params[:invoice_id]).presenter
```

Comment:
Hmm the `@order` instance variable is overwritten with a presenter? I know it's not caused by this PR but looks like a bad idea to me.

Reply:
The presenter imitates the behavior of the order on the new invoice template.
Is better to have a different variable that can represent the order or the presenter?

Reply:
I see, that makes sense 👍